.github/workflows/conda-win.yml

```diff
@@ -0,0 +1,71 @@
+name: Windows Conda End-to-end Test
+
+on:
+  push:
+    branches:
+      - main
+      - feature/windows_build
```

Comment:
Now that you've opened a PR, you can remove this (its actually double building right now.
We need to consolidate this file with the regular conda yaml.

Also it looks like the conda build is still using `vcpkg` for something (at the very least, it's downloading and building the libraries).

Reply:
Alternatively if you're getting tired of the build stuff (can't blame you) if you squash this we can split up the remaining tasks and do PRs into this PR.

Reply:
its tough with 100+ commits thats why a good squash helps out. 

Linux caching is fine on `main`, but it's limited to `10GB` and we're doing lots of full builds.

Reply:
while you're still pushing commits, I'll queue up some changes [over here](https://github.com/timkpaine/csp/pull/1#pullrequestreview-2060829481). When you hit a good stopping point, we can start to PR into this PR with some tweaks

Reply:
file deleted

Reply:
> I see my changes mixed in with these but now it will be tough with 100+ commits, so a good squash helps out.
> 
> Linux caching is fine on `main`, but total cache size is limited to `10GB` and we're doing lots of full builds, its possible even a single full build exceeds the cache size (which would mean a subsequent build won't use one or more of the cached artifacts).

cache is under 10GB now and it still looks like linux isnt using the vcpkg cache

Reply:
it does, i took screenshots on my personal computer of it being used.

Reply:
```
Post job cleanup.
Post job cleanup.
Cache hit occurred on the primary key vcpkg-Linux-X64-d86d4886a4[2](https://github.com/Point72/csp/actions/runs/9159293779/job/25179380322#step:20:2)f4f90d15411798f232205fd77d9701[3](https://github.com/Point72/csp/actions/runs/9159293779/job/25179380322#step:20:3)e8268d1f7e6d9fa105511f, not saving cache.
Post job cleanup.
ccache stats
Save cache using key "ccache-ccache-Linux-X64-".
/usr/bin/tar --posix -cf cache.tzst --exclude cache.tzst -P -C /home/runner/work/csp/csp --files-from manifest.txt --use-compress-program zstdmt
Failed to save: Unable to reserve cache with key ccache-ccache-Linux-X6[4](https://github.com/Point72/csp/actions/runs/9159293779/job/25179380322#step:20:4)-, another job may be creating this cache. More details: Cache already exists. Scope: refs/pull/229/merge, Key: ccache-ccache-Linux-X64-, Version: 1[5](https://github.com/Point72/csp/actions/runs/9159293779/job/25179380322#step:20:5)c80211763d03468c2c9070680654f9264282cf4daa2c6ceac80f2e3eaeb295
```

I think we're still just cache-thrashing, it does it FIFO

Reply:
![image](https://github.com/Point72/csp/assets/3105306/aaed2a12-2700-417a-af23-ab65bc36977c)

Reply:
we might need to delete and recreate the cache, it doesnt look like vcpkg liked the build assets:

`Restored 0 package(s) from /host/home/runner/vcpkg_cache in 668 us. Use --debug to see more details.`

Reply:
ok i think this is it:
https://github.com/Point72/csp/blob/5a6bb27fc4356ac76baf5592d20690269f6a2359/setup.py#L37

I must have missed that in my review, but we really shouldnt be doing that as i think it will conflict with the binary cache check in vcpkg

Reply:
Found the cause, fix on another PR